### DISCUSSION TITLE
An error occurred in trying to get the latest status details.

### DISCUSSION THREAD

**Discussion by SilberGecko6917** [upptime/upptime#1000] [General] [answered]
I tried to create a status page for my websites but then I got the error...

I then tried to go to the demo website and got exactly the same thing...

![grafik](https://github.com/user-attachments/assets/772aa6c1-8391-4d5f-81b5-1b9da9e2a3c6)
![grafik](https://github.com/user-attachments/assets/bf782ba2-bf0d-44b2-b600-f38f602e9d7b)

**Comment by AnandChowdhary** [comment, marked as answer]
More info in - https://github.com/upptime/upptime/issues/1002
